src/databricks/labs/ucx/hive_metastore/locations.py

```diff
@@ -194,6 +194,10 @@ def _external_locations(self) -> Iterable[ExternalLocation]:
                 external_locations.append(external_location)
                 continue
             queue.extend(curr.children.values())
+        if self._mounts_snapshot:
```

Comment:
Do all users want to add the root mount as external location? Or only the ones that have HMS federation enabled?

Reply:
Only one with HMS federation enabled. We make this determination when we crawl the mounts.